### DISCUSSION TITLE
Limit search results text?

### DISCUSSION THREAD

**Discussion by HarelM** [mkdocs/mkdocs#3482] [Q&A]
First and foremost, amazing product! I searched for something that I can use for our docs that is simple and powerfull and boy, this library is amazing!!

I'm using it in the following site:
https://maplibre.org/maplibre-gl-js/docs/

It works great except for one minor thing I'm not sure how to solve:
I have these html examples here: 
https://maplibre.org/maplibre-gl-js/docs/examples/

I basically wrote a small script that converts our testing examples to docs so they can be seen in the site, with an iframe and a code block, pretty straight forward.

The problem is that if I search for one of the examples I get the full code in the search results, which doesn't allow to see other results, this is even worse on mobile device:
![image](https://github.com/mkdocs/mkdocs/assets/3269297/c5c8369d-d54a-4227-acf3-9ab8a07c3ee3)

Is there any configuration I can use to solve this?

Here's my configuration:
https://github.com/maplibre/maplibre-gl-js/blob/main/mkdocs.yml

Thanks for all the amazing work done here!!!

**Comment by oprypin** [comment]
mkdocs-material replaces the search implementation, you can inquire there

**Reply by HarelM** [reply]
I see, thanks for the super quick response!
